test/unit/auth/token-generator.spec.ts

```diff
@@ -308,6 +308,28 @@ describe('FirebaseTokenGenerator', () => {
     tokenGenerator: new FirebaseTokenGenerator(new ServiceAccountSigner(cert), tenantId),
   }];
 
+  describe('Emulator', () => {
```

Comment:
We also need to test this via the Auth interface. Programmatically set the env variable and trigger the emulator mode.